{
  "api": "index",
  "file": "/test/free/data_stream/210_rollover_failure_store.yml",
  "name": "Lazily roll over a data stream's failure store after a shard failure",
  "origin": "yaml",
  "request": {
    "args": {
      "body": {
        "@timestamp": "2020-12-12",
        "count": "invalid value"
      },
      "index": "data-stream-for-rollover",
      "refresh": true
    }
  },
  "response": {
    "headers": {
      "content-length": "243",
      "content-type": "application/json",
      "location": "/.fs-data-stream-for-rollover-2025.10.07-000002/_doc/bPdUv5kBP-05coxOPoMf",
      "x-elastic-product": "Elasticsearch"
    },
    "payload": {
      "_id": "bPdUv5kBP-05coxOPoMf",
      "_index": ".fs-data-stream-for-rollover-2025.10.07-000002",
      "_primary_term": 1,
      "_seq_no": 0,
      "_shards": {
        "failed": 0,
        "successful": 1,
        "total": 2
      },
      "_version": 1,
      "failure_store": "used",
      "forced_refresh": true,
      "result": "created"
    },
    "statusCode": 201
  }
}